.sass-lint.yml

```diff
@@ -4,6 +4,8 @@ files:
     # Do not lint external files.
     - 'static/src/stylesheets/icons/**/*.scss'
     - 'static/src/stylesheets/email/foundation-emails/**/*.scss'
+    # Open issue: https://github.com/tonyganch/gonzales-pe/issues/197
```

Comment:
you ok to subscribe to https://github.com/tonyganch/gonzales-pe/issues/197 and remove this if it's fixed?

Reply:
Done 👍